<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Ishita" ignored="0" name="Command line test" tests="2" failures="2" timestamp="13 Jan 2018 14:29:13 GMT" time="122.625" errors="0">
  <testcase name="devidebyzero" time="12.166" classname="tests.TestSet1">
    <failure type="java.lang.AssertionError" message="expected [Cannot devide by 0] but found [Cannot divide by 0]">
      <![CDATA[java.lang.AssertionError: expected [Cannot devide by 0] but found [Cannot divide by 0]
	at tests.TestSet1.devidebyzero(TestSet1.java:44)
... Removed 40 stack frames]]>
    </failure>
  </testcase> <!-- devidebyzero -->
  <testcase name="multiply" time="21.487" classname="tests.TestSet1">
    <failure type="java.lang.AssertionError" message="Logged into system expected:&amp;lt;false&amp;gt; but was:&amp;lt;true&amp;gt;">
      <![CDATA[java.lang.AssertionError: Logged into system expected:<false> but was:<true>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at tests.TestSet1.multiply(TestSet1.java:33)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- multiply -->
</testsuite> <!-- Command line test -->
